bindings/python/mappings.py

```diff
@@ -1,18 +1,38 @@
 """Implements mappings between entities.
+
+Units are currently handled with pint.Quantity.  The benefit of this
+compared to explicit unit conversions, is that units will be handled
+transparently by mapping functions, without any need to specify units
+of input and output parameters.
+
+Shapes are automatically handled by expressing non-scalar quantities
+with numpy.
+
+TODO:
```

Comment:
Remove the two issue that have been done, and make a proper issue of the one missing.